request-api/requirements/requirements.in

```diff
@@ -0,0 +1,9 @@
+fastapi
```

Comment:
We should still pin versions of libraries we're using.   The benefit of using `pip freeze` to generate requirements is to pin the versions of transitive dependencies.